packages/vscode-extension/src/extension.ts

```diff
@@ -659,6 +664,12 @@ async function initializeContextKey(isTeamsFxProject: boolean) {
     "fx-extension.canUpgradeToArmAndMultiEnv",
     await canUpgradeToArmAndMultiEnv(workspaceUri?.fsPath)
   );
+
+  await vscode.commands.executeCommand(
+    "setContext",
+    "fx-extension.canAddFeature",
+    !isSPFxProject || isSPFxMultiTabEnabled()
```

Comment:
What features can users add if SPFx multi-tab is enabled?

Reply:
For support multi-tab feature, 'SPFx tab' will be released to allow users to add. When this is enabled, there'll be two options:  
1. SPFx  tab
2. CI/CD

Reply:
Updated, thanks!